vin/app/controllers/vin/notes_controller.rb

```diff
@@ -1,77 +0,0 @@
-class Vin::NotesController < ApplicationController
-respond_to :json
-
-  # GET /vin/sub/:uid/shipments/:sid/notes
-  def index
-
-    @subscriber = Subscriber.find(params[:sub_id])
```

Comment:
Since this seems to be recurring in this controller. you could have made it a `before_action` and DRY this up:
```ruby
before_action :find_subscriber

private

def find_subscriber
  @subscriber = Subscriber.find(params[:sub_id])
end
```